src/objects/set.rs

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) 2018 David Grunwald, Georges Racinet
```

Comment:
His name is `Daniel`, not `David`. :)

Reply:
Thanks, seems I've been sloppy when I decided that it wasn't right to put only my own for code so much similar to `dict.rs`.